apt-hook/Makefile

```diff
@@ -1,6 +1,9 @@
-all: hook
+all: build
 
-build: hook
+build: hook ubuntu-advantage.pot
+
+ubuntu-advantage.pot: hook.cc
+	xgettext hook.cc -o ubuntu-advantage.pot
```

Comment:
Excuse my ignorance here, but once the file is generated do we need to do something with it?

Reply:
Normally you'd ship the .pot and translated .po files in the tarball; but this is more of a hack, as we don't really care I guess, since we do translations on the package via langpacks anyway. So we can rely on launchpad picking the file up after the build and then shipping translations in langpacks.

Obviously, we'd want to have one pot file for python and C++ code, so that might need different work.